ui/DesignSystem/Component/Remote.svelte

```diff
@@ -1,15 +1,24 @@
 <script>
   import * as remote from "../../src/remote.ts";
 
+  import WithContext from "./WithContext.svelte";
+
   export let store = null;
+  export let context = null;
 </script>
 
 {#if $store.status === remote.Status.NotAsked}
   <slot name="not-asked" />
 {:else if $store.status === remote.Status.Loading}
   <slot name="loading" />
 {:else if $store.status === remote.Status.Success}
-  <slot data={$store.data} />
+  {#if context && typeof context === 'string'}
```

Comment:
Curious about this `typeof context === 'string'` guard - what are the other possibilities for the `context`, and why would we not use a `WithContext` wrapper for them?

Reply:
We use that prop to name the context variable provided, so we want to make sure it's only used as such, happy to see alternatives.